### ISSUE TITLE
Homepage bogging down because of counts?

### ISSUE THREAD

**Issue by rebeccacremona** [harvard-lil/perma#2908]
I think our DB is under-performing / I'm using it poorly. I'm getting zippy performance, using the app, but 504 for the homepage, and users have noticed.... I think these counts are slow: 
https://github.com/harvard-lil/perma/blob/develop/perma_web/perma/views/common.py#L64

Going to go ahead and remove them for now https://github.com/harvard-lil/perma/pull/2909

**Comment by rebeccacremona** [comment]
Ben suggests a database view might be the way to go here. To be investigated.

**Comment by jcushman** [comment]
We've run into slow counts in CAP -- see https://wiki.postgresql.org/wiki/Count_estimate

Are these already cached in a table somewhere? What's feeding https://perma.cc/stats ?

**Comment by rebeccacremona** [comment]
Yeah, I just switched to a count estimate for large tables in the Django admin! I've been thinking about whether we want to report total number of rows in the API too, or if we can switch to an estimate there, at least for certain kinds of queries.....

https://perma.cc/stats is indeed fed by some values cached in a table, but the table is, I believe, in disarray: it is populated via a signals and celery tasks in a way that I know I once discovered was lossy and/or somehow otherwise not particularly accurate, but I haven't dug in yet.... That's probably the project here: fix that data, and then use it in all the appropriate places.

**Comment by rebeccacremona** [comment]
See https://github.com/harvard-lil/perma/issues/2774
